Automattic/wp-calypso
# Update Newspack Blocks to v1.21.0

* Bumps Newspack Blocks to [v1.21.0](https://github.com/Automattic/newspack-blocks/releases/tag/v1.21.0)
* Includes a fix for https://github.com/Automattic/newspack-blocks/issues/700 (fix PR: https://github.com/Automattic/newspack-blocks/pull/701)
* Also includes a fix for PHPCS "unused variable" warnings: https://github.com/Automattic/newspack-blocks/pull/702

To test:

* In the `apps/editing-toolkit` folder, `yarn && yarn run build` to build changes and sync the new Blocks release.
* Load the changes on a sandbox and verify that https://github.com/Automattic/newspack-blocks/issues/700 is no longer reproducible.
* Load the changes on an Atomic site and verify that the issue is no longer reproducible there, either
